### PR TITLE
Update doc comment

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: parity-util-mem/src/malloc_size.rs [hunk 1]
```diff
@@ -585,6 +585,7 @@ impl<T: MallocSizeOf> MallocSizeOf for parking_lot::RwLock<T> {
 ///
 /// and when no "any: "
 ///
+/// ```rust
 /// use parity_util_mem::{malloc_size, malloc_size_of_is_0};
 ///
 /// struct Data<T> { pub T }
```
Comment (ordian) [inline parity-util-mem/src/malloc_size.rs]:
```suggestion
/// struct Data<T> { pub t: T }
```

Comment (NikVolf) [reply parity-util-mem/src/malloc_size.rs]: Oh I haven't realized it wasn't compiling before

**Comment by ordian** [comment]
```patch
diff --git a/parity-util-mem/src/malloc_size.rs b/parity-util-mem/src/malloc_size.rs
index 3bc9cc4..c68f8f6 100644
--- a/parity-util-mem/src/malloc_size.rs
+++ b/parity-util-mem/src/malloc_size.rs
@@ -588,12 +588,12 @@ impl<T: MallocSizeOf> MallocSizeOf for parking_lot::RwLock<T> {
 /// ```rust
 /// use parity_util_mem::{malloc_size, malloc_size_of_is_0};
 ///
-/// struct Data<T>(pub T)
+/// struct Data<T>(pub T);
 ///
 /// // generic argument (`T`) must be `impl MallocSizeOf`
 /// malloc_size_of_is_0!(Data<u8>);
 ///
-/// assert_eq!(malloc_size(&Data(0u8), 0);
+/// assert_eq!(malloc_size(&Data(0u8)), 0);
 /// ```
 #[macro_export]
 macro_rules! malloc_size_of_is_0(
```

**Comment by NikVolf** [comment]
> ```diff
> diff --git a/parity-util-mem/src/malloc_size.rs b/parity-util-mem/src/malloc_size.rs
> index 3bc9cc4..c68f8f6 100644
> --- a/parity-util-mem/src/malloc_size.rs
> +++ b/parity-util-mem/src/malloc_size.rs
> @@ -588,12 +588,12 @@ impl<T: MallocSizeOf> MallocSizeOf for parking_lot::RwLock<T> {
>  /// ```rust
>  /// use parity_util_mem::{malloc_size, malloc_size_of_is_0};
>  ///
> -/// struct Data<T>(pub T)
> +/// struct Data<T>(pub T);
>  ///
>  /// // generic argument (`T`) must be `impl MallocSizeOf`
>  /// malloc_size_of_is_0!(Data<u8>);
>  ///
> -/// assert_eq!(malloc_size(&Data(0u8), 0);
> +/// assert_eq!(malloc_size(&Data(0u8)), 0);
>  /// ```
>  #[macro_export]
>  macro_rules! malloc_size_of_is_0(
> ```

Thanks, already updated :)

**Pull request merged**